changelog/std-experimental-checkedint.dd

```diff
@@ -0,0 +1,28 @@
+New module `std.experimental.checkedint` defines `Checked`, a lightweight and
```

Comment:
The format is equal to git messages.
- first line = title
- empty line
- long description

For more details:
https://github.com/dlang/phobos/tree/master/changelog